pagmo2/include/pagmo/algorithms/GACO.hpp

```diff
@@ -68,15 +68,8 @@ class g_aco
     /// The log
     typedef std::vector<log_line_type> log_type;
```

Comment:
Giacomo,

I am not understanding your work flow. We made a pull request to your branch on Friday (https://github.com/Sceki/pagmo2/pull/2). Why you have not merged it or commented on it? 

Please merge that pull request and read carefully the text we wrote.

Reply:
Since you now made a further commit, it will likely create a mess of conflicts. I suggest you revert this commit and start from our PR.

Reply:
I am very sorry. I completely missed your PR (together with all your comments), apparently I did not notice the email, I will now go through it and revert this last commit.

Reply:
I did git revert <commit> . I hope this fixes the issue.

Reply:
Thank you very much for the thorough and clear comments. 

Apparently, there are still conflicts and I cannot accept your PR, even after I reverted the 'Code cleaning pt1'. Do you know what could cause the problem?

I am really sorry for this mess.

Reply:
I see no commit of the revert (look above, the last commit is Code Cleaning pt1. Did you forget to push?

Reply:
I indeed did not push. I just changed the HEAD of my branch to 'Update tests and algorithms folders (small fix)' and I pushed it.